src/strategies/kumainu/examples.json

```
@@ -0,0 +1,20 @@
+[
+  {
+    "name":"Example kuma inu query",
+    "strategy":{
+      "name":"kumainu",
+      "params":{
+        "decimals": 18,
+        "poolId": {
+          "kuma": "12",
+          "lp": "13"
+        }
+      }
+    },
+    "addresses": [
```

You need to define the network id here
```suggestion
    "network": "1",
    "addresses": [
```